packages/react-app/src/components/common/ConnectWeb3.jsx

```diff
@@ -1,12 +1,41 @@
-import { Button, Flex, Text } from '@chakra-ui/react';
+import { Button, ButtonGroup, Flex, Text } from '@chakra-ui/react';
 import { Web3Context } from 'contexts/Web3Context';
 import { WalletFilledIcon } from 'icons/WalletFilledIcon';
 import { FOREIGN_CHAIN_ID, HOME_CHAIN_ID } from 'lib/constants';
-import { getNetworkName } from 'lib/helpers';
+import { getNetworkName, getWalletProviderName } from 'lib/helpers';
+import { addChainToMetaMask } from 'lib/metamask';
 import React, { useContext } from 'react';
 
 export const ConnectWeb3 = () => {
-  const { connectWeb3, loading, account, disconnect } = useContext(Web3Context);
+  const {
+    connectWeb3,
+    loading,
+    account,
+    disconnect,
+    ethersProvider,
+  } = useContext(Web3Context);
+
+  const renderDisconnectButton = () => {
+    return getWalletProviderName(ethersProvider) === 'metamask' ? (
+      <ButtonGroup spacing={3} width="100%" mt={2}>
+        <Button onClick={disconnect} colorScheme="blue" px={12}>
+          Disconnect
+        </Button>
+        <Button
```

Comment:
As @akolotov has mentioned it would be good to have the button for both HOME_CHAIN_ID as well as FOREIGN_CHAIN_ID. 

Lets move the button from here to the above text :

To access OmniBridge, please switch to `xDai Chain` or `ETH Mainnet`

Here, the names of the networks can be replaced with buttons which switch the network onClick. 

We can also add a tooltip that says "Switch to xDai Chain" or "Switch to ETH Mainnet" upon hover.

Reply:
Here, maybe we can skip known ethereum networks `Kovan` and `ETH Mainnet` as metamask will already have them added by default. But definitely need to support 56=BSC, 77=SOKOL, and 100=XDAI